mautrix/bridge/portal.py

```diff
@@ -429,7 +429,9 @@ async def _send_message(
     ) -> EventID:
         if self.encrypted and self.matrix.e2ee:
             event_type, content = await self.matrix.e2ee.encrypt(self.mxid, event_type, content)
-        return await intent.send_message_event(self.mxid, event_type, content, **kwargs)
+        event_id = await intent.send_message_event(self.mxid, event_type, content, **kwargs)
+        asyncio.create_task(intent.mark_read(self.mxid, event_id))
```

Comment:
```suggestion
        if intent.api.is_real_user:
            asyncio.create_task(intent.mark_read(self.mxid, event_id))
```

Reply:
(implicit read receipts seem to work fine from the perspective of other users, it's just the unread count that's broken)